lightning/pkg/importer/import.go

```diff
@@ -1942,6 +1942,10 @@ func (rc *Controller) DataCheck(ctx context.Context) error {
 		}
 	}
 
+	if err := rc.checkSoureDataSize(ctx); err != nil {
```

Comment:
this check is cloud specific, i prefer we add those precheck in cloud service code and check it there. start a lightning instance, and then do precheck is much slower.